yarn.lock

```diff
@@ -7238,6 +7238,18 @@ nostr-tools@^1.17.0:
     "@scure/bip32" "1.3.1"
     "@scure/bip39" "1.2.1"
 
+nostr-tools@^1.17.0:
```

Comment:
There is an issue in yarn.lock - nostr-tools dependency is included twice (bad merge?) - could you fix this?

Reply:
Probably the easiest is just to revert this change and re-run yarn install.

Reply:
Done